dll/win32/browseui/CShellTaskScheduler.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * PROJECT:     ReactOS browseui
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     IShellTaskScheduler implementation
+ * COPYRIGHT:   Copyright 2020 Oleg Dubinskiy (oleg.dubinskij2013@yandex.ua)
+ */
+
+#pragma once
+
+class CShellTaskScheduler :
+    public CComCoClass<CShellTaskScheduler, &CLSID_ShellTaskScheduler>,
+    public CComObjectRootEx<CComMultiThreadModelNoCS>,
+    public IShellTaskScheduler
+{
+private:
+public:
+    CShellTaskScheduler();
+    ~CShellTaskScheduler();
+
+    // *** IShellTaskScheduler methods ***
+    STDMETHODIMP AddTask(IRunnableTask *pTask, REFGUID rtoid, DWORD_PTR lParam, DWORD dwPriority);
+    STDMETHODIMP RemoveTasks(REFGUID rtoid, DWORD_PTR lParam, BOOL fWaitIfRunning);
+    virtual UINT STDMETHODCALLTYPE CountTasks(REFGUID rtoid);
```

Comment:
Why not `STDMETHODIMP_(UINT)` instead of `virtual UINT STDMETHODCALLTYPE` ?

Reply:
Because I even didn't know that `STDDMETHODIMP` calling conversion support multiple data types. Sorry.
I'll follow up this in new PR then. :ok_hand:

Reply:
> Because I even didn't know that `STDMETHODIMP` calling conversion supports multiple data types. Sorry.
> I'll follow up this in new PR then. 👌

There is an underscore in that name, that is different from the name without the underscore.

Reply:
Ah, indeed. :sweat_smile: So this is a different conversion. Thanks.